util.go

```diff
@@ -120,16 +120,13 @@ func (b *Builder) copyFileWithTar(tarIDMappingOptions *IDMappingOptions, chownOp
 		}
 	}
 	return func(src, dest string) error {
+		var f *os.File
+
 		logrus.Debugf("copyFileWithTar(%s, %s)", src, dest)
-		f, err := os.Open(src)
+		fi, err := os.Lstat(src)
```

Comment:
Since we're doing the system.Lstat below at line 131, is this one required?

Reply:
system.Lstat seems like a portable way to retrieve the UID/GID, unfortunately it costs another stat as it cannot be used as a `FileInfo`